cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -257,4 +273,34 @@ max_tx_bytes = {{ .ChainCfg.Mempool.MaxTxBytes }}
 
 # Size of the cache (used to filter transactions we saw earlier) in transactions
 cache_size = {{ .ChainCfg.Mempool.CacheSize }}
+
+#######################################################
+###         State Sync Configuration Options        ###
+#######################################################
+[chain.statesync]
```

Comment:
Let's put a newline between the section heading and anything after, for readability.